web/spacewalk-web.changes

```diff
@@ -1,3 +1,4 @@
+- Refactor the panel.js components
```

Comment:
hum I would not add anything to the changes here. Last retrospective we discussed that only bugs or changes that are visible to the costumer should be presented here

Reply:
Yeah Maintenance team approached us asking to put more value to our changelog entries, in terms of "the changelog should be understandable by the customer, and should be deliver value to him". This PR instead, is about internal refactoring only.

Reply:
nope it just a refactor, I would remove it